collections/brandedUserProfile.js

```diff
@@ -359,7 +359,10 @@ BrandedUserSchema = new SimpleSchema({
     bringCar: fields.bool(o) // _internal.needsParking
   }),
   optional: true},
-  _phone: fields.string(o),
+  _phone: {
```

Comment:
Did you mean to remove the optionality?

Reply:
I didn't - I didn't realize that's what that `o` was indicating. I'm wondering if it _should_ be considered optional though. Thoughts?

Reply:
Yeah we should leave it optional. FB students for example don't have them

Reply:
Gotcha. Added this back in.